packages/firestore/package.json

```diff
@@ -56,6 +57,7 @@
     "mocha": "5.0.5",
     "npm-run-all": "4.1.2",
     "nyc": "11.6.0",
+    "prettier": "^1.12.1",
```

Comment:
This should be an explicit version (and as Gil points out, we should make sure it matches the root package.json version).